cabal-ghc901.project

```diff
@@ -24,6 +24,9 @@ packages:
          ./plugins/hls-module-name-plugin
          ./plugins/hls-ormolu-plugin
          ./plugins/hls-call-hierarchy-plugin
+
+with-compiler: ghc-9.0.1
+
```

Comment:
what does this do?

Reply:
It is the equivalent to do `-w ghc-9.0.1` or `--with-compiler ghc-9.0.1`, make the build use that compiler executable